src/facade/reply_builder.h

```diff
@@ -41,14 +68,19 @@ class SinkReplyBuilder {
   virtual ~SinkReplyBuilder() {
   }
 
+  static MGetStorage* AllocMGetStorage(size_t size) {
+    char* buf = new char[size + sizeof(MGetStorage)];
+    return new (buf) MGetStorage();
+  }
```

Comment:
Please check alignment 🤓 Even if placement new will accept non-aligned pointers, it might cut off a chunk. 

This is the easiest workaround I know: `new (std::align_val_t(alignof(Storage))) char[size]`

Reply:
i prefer to static_assert that MGetStorage has 8 byte alignment, which should work with any malloc implementation I know (all of fhem return pointers that at least 8 byte aligned).

Reply:
Yes, I know it's safe now, I just mentioned it in the generic case (it's hard to make it non 8 byte aligned except with long double, etc)